app/views/integrations/help.html.erb

```diff
@@ -8,6 +8,6 @@
 
 <ul>
   <li><a href="http://groups.google.com/group/TracksApp">Mailing list</a></li>
-  <li><a href="https://gitter.im/TracksApp/tracks">Gitter chat</a></li>
   <li><a href="https://webchat.freenode.net/#Tracks">IRC channel #Tracks@FreeNode</li>
+  <li><a href="https://matrix.to/#/#tracks:matrix.org">Matrix chat</a> (bridged to the IRC channel)</li>
```

Comment:
Since it's ultimately pointing to the same resource should this share the `li` with freenode, like in the README?

Reply:
Yeah, you're probably right. I first thought Matrix is a bit different from IRC and decided to retain the webchat link also, but I now changed both links to be on the same row for clarity.